py2app/recipes/ipython.py

```diff
@@ -3,4 +3,4 @@ def check(cmd, mf):
     if m is None or m.filename is None:
         return None
 
-    return {"includes": ['_sitebuiltins']}
+    return {"includes": ["_sitebuiltins"], "packages": ["IPython"]}
```

Comment:
IPython can be added to autopackages instead, _sitebuiltins is included automatically due to changes in py2app's site.py.